lib/entities/send_tax.rb

```diff
@@ -0,0 +1,19 @@
+module Entities
+  class SendTax < Tax
+    TAXES = {
+      USUAL_TYPE: 0.05,
+      CAPITALIST_TYPE: 0.1,
+      VIRTUAL_TYPE: 0.88
+    }.freeze
```

Comment:
Do we need TAXES here? Check WithdrawTax and other tax classes. I see different implementation of the similar logic